### PR TITLE
Bug 17749: Show version information in "About" window

### PR DESCRIPTION
**Pull request by azadi**
(As we decided to review patches before submitting to master, here goes.)

This shows the version information in the about window.

@boklm @arlolra

### CODE REVIEW

**Comment by arlolra** [comment]
Can you post a screenshot?

**Comment by azadi** [comment]
Text suggestions welcome. 
tormessenger

File: projects/instantbird/branding/credits.patch [hunk 1]
```diff
@@ -167,9 +178,15 @@ diff --git a/im/content/aboutDialog.xul b/im/content/aboutDialog.xul
 diff --git a/im/locales/en-US/chrome/instantbird/aboutDialog.dtd b/im/locales/en-US/chrome/instantbird/aboutDialog.dtd
 --- a/im/locales/en-US/chrome/instantbird/aboutDialog.dtd
 +++ b/im/locales/en-US/chrome/instantbird/aboutDialog.dtd
+@@ -1,4 +1,4 @@
+-<!ENTITY aboutDialog.title      "About &brandFullName;">
++<!ENTITY aboutDialog.title      "About &brandShortName; &tmVersion;">
```
Comment (arlolra) [inline projects/instantbird/branding/credits.patch]: Think I prefer it w/o `&tmVersion;` here.

**Comment by arlolra** [comment]
How about?

```
Tor
MESSENGER

Version &tmVersion; (based on Instantbird IM &ibVersion)

&#169;2015-&copyrightYear; The Tor Project, Inc. This program is free software; you can redistribute it and/or modify it under the terms of the GNU GPL license version 2.0 or later.
```

`&ibVersion` can be "Nightly" to start.

**Comment by arlolra** [comment]
Btw, you know you can `--force` push to the same branch to update the pull request, right? Just mentioning since I noticed this is pull #2 and it'd be nice to preserve the discussion as we iterate. Thanks.

**Comment by azadi** [comment]
I will update the patch.

(It's pull #2 on purpose. It will be sad if I created pull requests every single time :)

**Comment by azadi** [comment]
a

Better?

**Comment by arlolra** [comment]
Can we get rid of the " - Beta" from the dialog title? It's redundant. Also, I wonder if pushing the "(based on ...)" to a newline (and maybe capitalizing the "b") will look a little cleaner? Thoughts?

**Comment by azadi** [comment]
OK with removing -beta from the title. I am not so sure about having "Based on Instantbird Nightly" on a line by itself. Perhaps "Based on the Instantbird IM client (Nightly)"?

(Yayay, bikeshedding!)

**Comment by arlolra** [comment]
I was thinking,

```
Version xxx
(Based on Instantbird IM Nightly)
```

**Comment by azadi** [comment]
ok

**Comment by arlolra** [comment]
Can we get the version and "(Based on" lines a little tighter together and centred?

**Comment by azadi** [comment]
Do you want both version and "based on" centered with respect to the logo? (OK with making them tighter).

**Comment by arlolra** [comment]
Yes

**Comment by azadi** [comment]
2016-02-17-160323_319x492_scrot

**Comment by arlolra** [comment]
Almost there!

Can there be no space between Version and Based on? The next line.

**Comment by azadi** [comment]
2016-02-17-164452_336x495_scrot

Out of paint!

**Comment by arlolra** [comment]
LGTM. Push it!

**Pull request closed without merging**